src/test/common/utils/interest.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai'
+import { getInterestDetails } from 'common/interest'
+import * as claimStoreServiceMock from '../../http-mocks/claim-store'
+import { Claim } from 'claims/models/claim'
+import { InterestType } from 'claim/form/models/interest'
+import { Moment } from 'moment'
+import { MomentFactory } from 'common/momentFactory'
+
+describe('getInterestDetails', () => {
+
+  it('should return undefined when interest has not been selected', async () => {
+    let claim: Claim = new Claim().deserialize(claimStoreServiceMock.sampleClaimObj)
```

Comment:
Plase use const.

Plase also do not relay on `claimStoreServiceMock.sampleClaimObj`, that makes this test unnecessarily dependent on mocked test data. Prepare test data specifically for that tests case.

Reply:
which is what he was doing?
I would encourage reverting this change, as this makes it a maintenance nightmare if every test fully makes its own test data.
Either:
1. Use existing base sample data, like you had before and like we had in the claim store
2. or use absolute minimum amount of data required.
and don't just copy the claim wholesale in